src/SdkCommon/AppAuthentication/Azure.Services.AppAuthentication.IntegrationTests/Microsoft.Azure.Services.AppAuthentication.IntegrationTests.csproj

```diff
@@ -4,10 +4,14 @@
         <TargetFrameworks>net472;net452;net461;netcoreapp2.0;netcoreapp1.1</TargetFrameworks>
     </PropertyGroup>
 	<PropertyGroup Condition="'$(TargetFramework)' == 'net472'">
-		<DefineConstants>net472;FullNetFx</DefineConstants>		
-		<OutputPath>bin\$(Configuration)\</OutputPath>
-		<DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
+        <DefineConstants>net472;FullNetFx</DefineConstants>		
+        <OutputPath>bin\$(Configuration)\</OutputPath>
+        <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
 	</PropertyGroup>
+    <PropertyGroup>
+        <ExcludeFromBuild/>
```

Comment:
Did you intend to set this property to empty?

Reply:
Yes, again this was one of the test cases that I was using as and secondly a way to advertise these properties outside the documentation.

Reply:
I'd suggest that you set the value explicitly to false here then to make it clear, as it isn't clear what empty would mean.